source/Cosmos.System2/Kernel.cs

```diff
@@ -138,14 +138,18 @@ public virtual void Start()
         /// 4. IDE initialisation, true/false, default: true
         /// If you need anything else to be initialised really early on, place it here.
         /// </summary>
-        protected virtual void OnBoot() {
+        protected virtual void OnBoot()
+        {
```

Comment:
Should be checked-out.

Reply:
+1. Please revert all the changes to this file

Reply:
> +1. Please revert all the changes to this file

I am alredy reveted the canges